### PR TITLE
implemented func "shop_trip"

### PR DESCRIPTION
**Pull request by 380974261314kv**

### CODE REVIEW

**Comment by 380974261314kv** [comment]
Can I use suck a check:
if not ".0" in str(total_product_cost)

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,81 @@
...
+            purchase_time = date.strftime("%d/%m/%Y %H:%M:%S")
+            print(f"Date: {purchase_time}")
+            print(f"Thanks, {customer.name}, for your purchase!")
+            print("You have bought:")
+            total_price = 0
+            for product, quantity in customer.product_cart.items():
+                product_price = cheapest_shop.products.get(product, 0)
+                total_product_cost = product_price * quantity
+                total_price += total_product_cost
+                # if total_product_cost.is_integer():
```
Comment (vasylhnatiuk) [inline app/main.py:67]: delete useless comment

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,81 @@
...
+                if ".0" not in str(total_product_cost):
+                    print(f"{quantity} {product}s"
+                          f" for {round(total_product_cost, 1)} dollars")
+                else:
+                    print(f"{quantity} {product}s "
+                          f"for {int(total_product_cost)} dollars")
+            print(f"Total cost is {round(total_price, 2)} dollars")
+            print("See you again!\n")
+            print(f"{customer.name} rides home")
```
Comment (vasylhnatiuk) [inline app/main.py:76]: when cutomer rides home him location must equal home location

File: app/main.py [hunk 3]
```diff
@@ -1,3 +1,81 @@
...
+
+def calculate_fuel_cost(
+        distance: float,
+        fuel_price: float,
+        fuel_consumption: float
+) -> float:
+    return (distance / 100) * fuel_consumption * fuel_price
+
+
+def shop_trip() -> None:
```
Comment (vasylhnatiuk) [inline app/main.py:24]: it is really big function. try to seperate it to a few functions/methods with different responsibilities

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**